root/static/scripts/edit/components/FormRowNameWithGuessCase.js

```diff
@@ -16,9 +16,9 @@ import GuessCase from '../../guess-case/MB/GuessCase/Main.js';
 import FormRowText from './FormRowText.js';
 import {
   type ActionT as GuessCaseOptionsActionT,
-  type StateT as GuessCaseOptionsStateT,
   createInitialState as createGuessCaseOptionsState,
   runReducer as runGuessCaseOptionsReducer,
+  type StateT as GuessCaseOptionsStateT,
 } from './GuessCaseOptions.js';
```

Comment:
It’s pretty annoying to not have a stable sorting rule. I would prefer to stick with a version of `eslint-plugin-simple-import-sort` that allows sorting as before. There is no issue with sticking to an older but more appropriate version of a linter used by developers only.
In the future maybe we should consider switching (back?) to `eslint-plugin-import` which supports more options.

Reply:
Well, the point of this import plugin is that it always allows just running `--fix` to sort all imports, so it's meant as a "let it do it automatically" tool, which did not work with the previous tool IIRC. That said, if we prefer the old sort, we can just stick to it, I don't think there were any significant bugfixes or whatnot in the intermediate versions anyway :)

Reply:
https://github.com/lydell/eslint-plugin-simple-import-sort/issues/124 contains the context for this change. Personally, I find keeping the type imports together much more readable, and we don't require any compatibility with the cited tools (TypeScript or dprint). I wish it had been made configurable, but the goal of the project is simplicity.

So I'd agree with sticking with the previous version for now. As an aside, https://github.com/import-js/eslint-plugin-import/ claims to support `--fix`.

Reply:
Ok! Dropped this commit :)